doc/upgrading.md

```diff
@@ -19,6 +20,22 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 
 ## Version 32.0
 
+* [RMB-609](https://issues.folio.org/browse/RMB-609) Upgrade to Vert.x 4:
+  * Replace deprecated `io.vertx.core.logging.Logger` by
+    `org.apache.logging.log4j.Logger`, for example
+    `private static final Logger LOGGER = LogManager.getLogger(Foo.class)`
+    (using `getLogger()` without argument requires Multi-Release for
+    stack walking, see below).
+  * Replace `HttpClient` `.getAbs`, `.postAbs`, `.putAbs`, `.deleteAbs` by
+    `WebClient.requestAbs(HttpMethod method, ...)`.
+  * Replace `Future.setHandler` by `Future.onComplete`.
+  * `Future` has been split into `Future` and `Promise`, see
+    [Futurisation in Vert.x 4](futurisation.md).
+  * In order to have structured logging for tests RMB based modules should copy [log4j2-test.properties](https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/src/test/resources/log4j2-test.properties) into their `/src/test/resources` directory
+  * The responseHandler function in ClientGenerator should be migrated from Handler<HttpClientResponse> to Handler<AsyncResult<HttpResponse<Buffer>>>
```

Comment:
RMB based modules usually call ClientGenerator from pom.xml. One examle is mod-configuration. I don't see a change in https://github.com/folio-org/mod-configuration/pull/102/files , can you be more verbose what developers of RMB based modules should change?

Reply:
done